packages/gluestick/src/renderer/main.js

```diff
@@ -136,24 +120,11 @@ module.exports = function startRenderer({ config, logger }: Context) {
 
     readAssets(assetsFilename)
       .then((assets: Object): Promise<void> => {
-        return middleware(
-          { config, logger },
-          req,
-          res,
-          { entries, entriesConfig, entriesPlugins: runtimePlugins },
```

Comment:
note the crazy renaming here! entriesPlugins is really runtimePlugins. hooksHelper is really hooksHelper.call. Knowing this will help the middleware.js changes make more sense.